docs/_layouts/main.html

```diff
@@ -8,11 +8,11 @@
 
 {% assign sidenav = site.data.nav[page.sidenav] | default: page.sidenav %}
 
-<main class="usa-layout-docs usa-section" id="main-content">
+<main class="usa-section" id="main-content">
   <div class="grid-container">
     {% if sidenav %}
       <div class="grid-row grid-gap">
-        <aside class="usa-layout-docs__sidenav desktop:grid-col-3">
+        <aside class="display-none tablet:display-block tablet:grid-col-3">
```

Comment:
These changes move away from the internal-ish [`usa-layout-docs` styles](https://github.com/uswds/uswds/blob/develop/packages/usa-layout-docs/src/styles/_usa-layout-docs.scss), which should have an accessibility benefit by avoiding [issues concerning `order` styles](https://developer.mozilla.org/en-US/docs/Web/CSS/order#accessibility_concerns). Instead, the markup is duplicated so that it still appears below content as expected at small screens, but reflecting that in the markup as well.

Reply:
What's our timeline for spinning down this site? Should we just start removing content from it so we don't have to maintain it?

Reply:
> What's our timeline for spinning down this site? Should we just start removing content from it so we don't have to maintain it?

Still nailing down specifics on this. #333 will prep the production site to 404. Once that's in place, I was going to remove many of the non-component content pages, but may want to continue keeping around some of the rest of the documentation for internal development reference, pull request preview links, and visual regression specs. So it'd continue to be useful to have this working, at least for the short term.